contributors/devel/sig-testing/testing-strategy.md

```diff
@@ -26,6 +26,39 @@ The Kubernetes job uses [prow](https://prow.k8s.io) to implement the CI system.
 - **Postsubmit:** Runs after code is merged. Useful for building artifacts.
 - **Periodic:** Runs at scheduled intervals. Ideal for monitoring trends and catching regressions.
 
+#### Non-blocking triggered by path
+
+Usually, blocking pre-submit jobs run by default and non-blocking jobs don't. The `/test` command
+has to be used explicitly for such non-blocking jobs. It is possible to configure such
+jobs so that they [run automatically when certain paths are modified](https://github.com/kubernetes/test-infra/blob/ee70308f09c10f7cd933c26c98acc7ebf785d436/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L3201-L3202).
+
+Use this with care! A non-blocking job that fails confuses other contributors
```

Comment:
Consider making this a warning or other callout? (https://github.com/orgs/community/discussions/16925)

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

Reply:
Done.